### PR TITLE
Begin setup for Github actions based CI

### PR DESCRIPTION
**Pull request by philipbelesky**

### CODE REVIEW

File: tabbycat/settings/github.py [hunk 1]
```diff
@@ -1,14 +1,14 @@
-# ==============================================================================
+ ==============================================================================
 # Travis CI
```
Comment (tienne-B) [inline tabbycat/settings/github.py:2]: Not here?

File: .github/README.md [hunk 2]
```diff
@@ -7,7 +7,7 @@
 [![Release](https://img.shields.io/github/release/tabbycatdebate/tabbycat.svg)](https://github.com/tabbycatdebate/tabbycat/releases)
 [![Crowdin](https://badges.crowdin.net/tabbycat/localized.svg)](https://crowdin.com/project/tabbycat)
 [![Docs](https://readthedocs.org/projects/tabbycat/badge/)](http://tabbycat.readthedocs.io/en/stable/)
-[![Build Status](https://travis-ci.org/TabbycatDebate/tabbycat.svg?branch=develop)](https://travis-ci.org/TabbycatDebate/tabbycat)
+![example workflow name](https://github.com/TabbycatDebate/tabbycat/workflows/django/badge.svg)
```
Comment (tienne-B) [inline .github/README.md:10]: Could have kept the "Build Status" alt-text.

Comment (philipbelesky) [reply .github/README.md:10]: Yup just fixing that now - the badge itself is also not working

**Comment by czlee** [comment]
🥳 Thanks for figuring this out!

**Pull request merged**